website/docs/r/spring_cloud_application_insights_application_performance_monitoring.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "Spring Cloud"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_application_insights_application_performance_monitoring"
+description: |-
+  Manages a Spring Cloud Application Insights Application Performance Monitoring.
+---
+
+# azurerm_spring_cloud_application_insights_application_performance_monitoring
+
+-> **NOTE:** This resource is applicable only for Spring Cloud Service with enterprise tier.
+
+Manages a Spring Cloud Application Insights Application Performance Monitoring.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  application_type    = "web"
+}
+
+resource "azurerm_spring_cloud_service" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "E0"
+}
+
+resource "azurerm_spring_cloud_application_insights_application_performance_monitoring" "example" {
+  name                         = "example"
+  spring_cloud_service_id      = azurerm_spring_cloud_service.example.id
+  connection_string            = azurerm_application_insights.example.instrumentation_key
+  globally_enabled             = true
+  role_name                    = "test-role"
+  role_instance                = "test-instance"
+  sampling_percentage          = 50
+  sampling_requests_per_second = 10
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Spring Cloud Application Insights Application Performance Monitoring. Changing this forces a new resource to be created.
+
+* `spring_cloud_service_id` - (Required) The ID of the Spring Cloud Service. Changing this forces a new resource to be created.
+
+---
+
+* `globally_enabled` - (Optional) Specifies whether the Spring Cloud Application Insights Application Performance Monitoring is enabled globally. Defaults to `false`.
+
+* `connection_string` - (Optional) The connection string of the Application Insights resource.
+
+* `role_name` - (Optional) The name of the role.
+
+* `role_instance` - (Optional) The name of the role instance.
+ 
+* `sampling_percentage` - (Optional) The percentage of requests to sample.
+
+* `sampling_requests_per_second` - (Optional) The number of requests per second to sample. 
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Spring Cloud Application Insights Application Performance Monitoring.
```

Comment:
change to "The ID of the Spring Cloud Application Performance Monitoring resource for Application Insights"